examples/dreambooth_custom_character/README.md

```diff
@@ -0,0 +1,200 @@
+
+```python
+# Run inference with the base stable diffusion model 
+#OLD: pipe = StableDiffusionPipeline.from_pretrained("CompVis/stable-diffusion-v1-4")
+#NEW: 
+pipe = StableDiffusionPipeline.from_pretrained("<your-hf-username>/stable-diffusion-oxified")
+```
+
+### Results
+Let’s compare the models using the same 3 prompts we applied pre-Dreambooth.
+
+![](images/happy-fluffy-after.png)
+![](images/oxenai-ox-after.png)
+![](images/plain-ox-after.png)
+
+
+
+Lots of cool differences across these three prompting strategies to unpack here! Some key takeaways: 
+
+1. The model has become shockingly good at producing new observations in our target ox style given how few training observations it has seen. This is the core competence of Dreambooth, and we're really impressed with how it turned out! 
+2. As expected, the model is best at recalling our specific ox style when we use our training token, `an image of the oxenai ox`. In the third prompt, where we reference just `an ox`, the model’s underlying understanding of what an ox looks like from its original training data still shines through (though conditioned slightly by our examples—see the white fur and prominent horns). 
+3. Even without explicit mention of the `oxenai ox` token, this fine-tuning has made the model more capable of producing cartoonish, fuzzy oxen—just look at the gap in coherence between the pre- and post-Dreambooth results for the first prompt!
+
+### What’s next?
+
+Though these early results are promising, the model still struggles a bit in domains dramatically different from the fine-tuning images.
+![4 rendered images in response to the prompt "an image of the oxenai ox surfing", in which three are just humans and only one appears to be a cartoon ox](images/surfing-fail.png)
+
+Additionally, it’s nailed the general characteristics of our desired ox (cute, cartoonish, fuzzy) and is showing much-improved consistency, but isn’t yet able to produce a singular, recognizable character. 
+
+As an attempt to solve both these issues, we’re going to use this new model (and an [Oxen-powered RLHF interface](https://blog.oxen.ai/collecting-data-from-human-feedback-for-generative-ai/)) to gain further, prompt-specific feedback for additional fine-tuning. We think that this will be key in bridging the gap to a generalizable, consistent character—stay tuned for the results! 
+
+At OxenAI we want to see what you are building! Reach out at hello@oxen.ai, follow us on Twitter [@oxendrove](https://twitter.com/oxendrove), dive deeper into the [documentation](https://github.com/Oxen-AI/oxen-release), or **Sign up for Oxen today. http://oxen.ai/register.**
+
+And remember—for every star on [GitHub](https://github.com/Oxen-AI/oxen-release), an ox gets its wings.
+
+[No, really...go check it out!](https://oxen.ai/ox/FlyingOxen) 
+
+![An AI-generated image of a purple, magestic, winged space ox](images/winged-ox.png)
```

Comment:
can we choose a slightly more wing-y ox lol this one looks like the wings aren't attached to it's body